mlflow/artifacts/__init__.py

```diff
@@ -57,3 +62,108 @@ def download_artifacts(
     artifact_repo = get_artifact_repository(artifact_uri)
     artifact_location = artifact_repo.download_artifacts(artifact_path, dst_path=dst_path)
     return artifact_location
+
+
+def load_text(artifact_uri: str) -> str:
+    """
+    Loads content of the artifact file as python str object
+    :param artifact_uri: artifact location
+    :return: str
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+        with mlflow.start_run() as run:
+            artifact_uri = run.info.artifact_uri
+            mlflow.log_text("This is a sentence", "file.txt")
+            file_content  = mlflow.artifacts.load_text(artifact_uri + "/file.txt")
+            print(file_content)
+
+    .. code-block:: text
+        :caption: Output
+
+        This is a sentence
+    """
+    local_artifact = download_artifacts(artifact_uri)
```

Comment:
```suggestion
    with tempfile.TemporaryDicrectory() as tmpdir:
        local_artifact = download_artifacts(artifact_uri, dst_path=tmpdir)
        with open(local_artifact) as local_artifact_fd:
            ...
```

Can we use `TemporaryDicrectory` to immediately remove the downloaed file after reading its content?

Reply:
good point ! Have modified code accordingly :)